styles.css

```diff
@@ -0,0 +1,265 @@
+    margin: 6px 0;
+}
+
+.bottom_part_of__the_burger {
+
+    width: 24px;
+    height: 0px;
+    bottom: 20px;
+    margin: 6px 0;
+    border: 1px solid #000000;
+
+}
+
+
+
+
+.about_blog {
+    font-family: 'New York Extra Large', sans-serif;
+    font-size: 34px;
+
+    height: 172px;
+    font-weight: 700;
+    font-style: normal;
+    line-height: 120%;
+    text-align: center;
+    margin: 32px 20px 0px 20px;
+    vertical-align: top;
+
+
+
+}
+
+.small_text_about_blog {
+    font-family: 'SF Mono', sans-serif;
+    font-size: 16px;
+
+    height: 81px;
+    
+    line-height: 150%;
+    text-align: center;
+    vertical-align: top;
+    margin: 24px 20px 0px 20px;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+
+
+}
+
+.line_of_first_page {
+
+    height: 0px;
+    border: 2px solid #000000;
+    margin: 52px 0 0 0;
+}
+
+.articles {
+    font-family: 'New York Extra Large', sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    width: 164px;
+    white-space: nowrap;
+    height: 36px;
+    line-height: 100%;
+    margin: 56px 100px 0px 100px;
+    vertical-align: top;
+}
+
+.second_page_photos_first {
+    margin: 29px 36px 0px 35px;
+
+}
+
+.second_page_photos {
+    margin: 42px 36px 0px 35px;
+   
+}
+
+
+.text_after_images {
+    font-family: 'SF Pro Display', sans-serif;
+    font-style: normal;
+    
+    font-size: 22px;
+   
+    line-height: 120%;
+    text-align: center;
+    vertical-align: top;
+    font-weight: 450;
+    margin: 20px 56px 0px 56px;
+}
+.text_after_images_margins{
+    font-family: 'SF Pro Display', sans-serif;
+    font-style: normal;
+   
+    font-size: 22px;
+    line-height: 26px;
+    text-align: center;
+    vertical-align: top;
+    font-weight: 450;
+    height: 104px;
+    margin: 20px 46px 0px 46px;  
+}
+.standart_height78 {
+    height: 78px;
+}
+
+.hello_world {
+    height: 52px;
+}
+.modern {
+
+    height: 104px;
+}
+
+.black_square {
+    background-color: #000000;
+    color: #FFFFFF;
+    height: 456px;
+    margin-top: 87px;
+
+    overflow: hidden;
+    position: relative;
+}
+
+.square_first_text {
+    font-family: 'SF Pro Display', sans-serif;
+    width: 1024px;
+    display: flex;
+    justify-content: space-between;
+    position: absolute;
+    margin: 22px 0px 0px 0px;
+    gap: 13px;
+    font-size: 21px;
+    white-space: nowrap;
+    line-height: 28px;
+    transform: translateX(-30%);
+
+
+
+}
+
+.bold_text_center{
+    font-weight: bold; 
+    text-align: center;
+}
+
+.text_сenter {
+    text-align: center;
+}
+
+
+
+
+.bottom_rose {
+    margin: 153px 65px 0px 65px;
+    margin-top: 153px;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+
+}
+
+.lorem {
+    font-family: 'New York Small';
+    font-style: normal;
+    font-size: 13px;
+    line-height: 140%;
+    margin: 26px 44px 0px 44px;
+
+}
+
+
+
+.links {
+    display: flex;
+    justify-content: space-between;
+    width: 181px;
+    margin: 32px 97px 0px 97px;
```

Comment:
I recommend you to read about [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) in more detail and understand how to use it to center your elements on the page